### PR TITLE
Task ys/j86 attach audio to episode functionality

### PR DESCRIPTION
**Pull request by yaroslav207**

### CODE REVIEW

File: backend/src/api/episodes/episodes.api.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Router } from 'express';
 import {
-  episode as EpisodeValidationSchema,
+  episodeCreate as EpisodeValidationSchema,
```
Comment (ghost) [inline backend/src/api/episodes/episodes.api.ts]:
```suggestion
  episodeCreate as episodeCreateValidationSchema,
```

File: backend/src/common/enums/episode/episode-type.enum.ts [hunk 2]
```diff
@@ -1 +1,5 @@
-export { EpisodeType } from 'jabber-shared/common/enums/enums';
```
Comment (ghost) [inline backend/src/common/enums/episode/episode-type.enum.ts:1]: rename `backend/src/common/enums/episode/episode.ts`

File: frontend/src/common/types/episode/episode-form-payload.type.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { EpisodePayloadKey } from 'common/enums/enums';
+import { EpisodePayload } from 'jabber-shared/common/types/types';
+
+type EpisodeFormPayload = EpisodePayload & {
+  [EpisodePayloadKey.PODCAST_ID]: number;
```
Comment (ghost) [inline frontend/src/common/types/episode/episode-form-payload.type.ts]:
```suggestion
```
let's set it in the action

File: frontend/src/components/configurate-episode/components/create-episode-form/common/constants.ts [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
-const DEFAULT_CREATE_EPISODE_PAYLOAD: EpisodeCreatePayload = {
-  [EpisodeCreatePayloadKey.NAME]: '',
-  [EpisodeCreatePayloadKey.DESCRIPTION]: '',
-  [EpisodeCreatePayloadKey.USER_ID]: DEFAULT_USER_ID,
-  [EpisodeCreatePayloadKey.PODCAST_ID]: DEFAULT_PODCAST_ID,
-  [EpisodeCreatePayloadKey.TYPE]: EpisodeType.PUBLIC,
+const DEFAULT_CREATE_EPISODE_PAYLOAD: EpisodeFormPayload = {
+  [EpisodePayloadKey.NAME]: '',
+  [EpisodePayloadKey.DESCRIPTION]: '',
+  [EpisodePayloadKey.PODCAST_ID]: DEFAULT_PODCAST_ID,
```
Comment (ghost) [inline frontend/src/components/configurate-episode/components/create-episode-form/common/constants.ts]:
```suggestion
```

File: frontend/src/components/configurate-episode/components/create-episode-form/create-episode-form.tsx [hunk 5]
```diff
@@ -21,36 +28,45 @@ const CreateEpisodeForm: React.FC<Props> = ({ onSubmit }) => {
     dataStatus: episode.dataStatus,
   }));
 
+  const acceptExtension = getFileExtensions(
```
Comment (ghost) [inline frontend/src/components/configurate-episode/components/create-episode-form/create-episode-form.tsx]: move it from the component

File: shared/src/common/types/record/record-create-payload.type.ts [hunk 6]
```diff
@@ -4,6 +4,7 @@ type RecordCreatePayload = {
   [RecordCreatePayloadKey.FILE_URL]: string;
   [RecordCreatePayloadKey.FILE_SIZE]: number;
   [RecordCreatePayloadKey.EPISODE_ID]: number;
+  [RecordCreatePayloadKey.PUBLIC_ID]: string;
```
Comment (ghost) [inline shared/src/common/types/record/record-create-payload.type.ts:7]: good catch 👍

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by yevhenprybytok — DISMISSED** [review]

File: frontend/src/validation-schemas/episode/create-episode/create-podcast.validation-schema.ts [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import { Joi } from 'helpers/helpers';
```
Comment (qwiz1) [inline frontend/src/validation-schemas/episode/create-episode/create-podcast.validation-schema.ts]: frontend/src/validation-schemas/episode/create-episode/create-podcast.validation-schema.ts
This is probably a mistake in the schema file name. I think there must be:
frontend/src/validation-schemas/episode/create-episode/create-episode.validation-schema.ts

**Review by Denidzio — DISMISSED** [review]

**Review by ghost — APPROVED** [review]
👍

**Pull request merged**